packages/logrotate/build

```diff
@@ -10,6 +10,12 @@ make
 make install
 
 
+# Create the module file
+module="$PKG_PATH/dcos.target.wants_slave/logrotate.json"
```

Comment:
dcos.target.wants is for systemd units which should be started. `etc` for config.

The `_master`, `_slave` and friends are pkgpanda stuff, what that means is "only use the files in this folder if the machine has that particular role". Primarily used to select a set of services to only start on masters, or only start on slaves, rather than to start on every machine.